<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportStl</class>
 <widget class="QDialog" name="ExportStl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GB_FileFormat">
     <property name="title">
      <string>STL File Format</string>
     </property>
     <widget class="QRadioButton" name="RB_AsciiFormat">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>32</y>
        <width>52</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ascii</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="RB_BinaryFormat">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>61</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Bi&amp;nary</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Resolution</string>
     </property>
     <widget class="QComboBox" name="CB_STLResolution">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>79</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="GB_HalfObject">
     <property name="title">
      <string>Half Ob&amp;ject</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="RB_XYPlane">
        <property name="text">
         <string>&amp;XY Plane (Horizontal simmetry)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RB_XZPlane">
        <property name="text">
         <string>X&amp;Z Plane (Vertical simmetry)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportStl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportStl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
